### PR TITLE
Add a PR template

### PR DESCRIPTION
**Pull request by tbedford**
> NOTE: To reviewers - please do not review PRs in the DRAFT state, as the PR may change substantially before ready to review. Thanks.                                                                                                       
                                                                                                                       
## Description                                                                                                         
                                                                                                                       
A PR description indicating the purpose of the PR.                                                                     
                                                                                                                       
* [JIRA ticket](link).                                                                                                 
                                                                                                                       
## Review                                                                                                              
                                                                                                                       
Instructions on how to review the PR.                                                                                  
                                                                                                                       
* [Page to review](link)

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+> NOTE: To reviewers - please do not review PRs in the DRAFT state, as the PR may change substantially before ready to review. Thanks.
```
Comment (marklewin) [inline pull_request_template.md:1]:
```suggestion
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.
```

**Review by marklewin — CHANGES_REQUESTED** [review]
I'm just wondering if we can make the NOTE a bit harder to ignore by using **bold**, etc?

**Review by marklewin — APPROVED** [review]
LGTM!

**Review by m-hulbert — APPROVED** [review]
LGTM 👍

**Pull request merged**